lib/cartopy/mpl/geoaxes.py

```diff
@@ -1368,6 +1374,7 @@ def gridlines(self, crs=None, draw_labels=False,
                   xlocs=None, ylocs=None, dms=False,
                   x_inline=None, y_inline=None, auto_inline=True,
                   xformatter=None, yformatter=None, xlim=None, ylim=None,
+                  rotate_labels=True, inside=False,
```

Comment:
It is hard for me to understand this choice. This would mean that the default mercator plot is the second one.
![gridliner_default_rotation](https://user-images.githubusercontent.com/1941408/96635539-44d8c100-131c-11eb-94e9-e14ae2e05b97.png)

Reply:
Hm... Good point. I agree that 2 is not a good default choice! I guess, my basic preference is to limit the jumping around of angles when I'm panning around/zooming in on the map. This is a pretty minor detail and personal preference really. Would `rotate_labels=False` be better/worse than `True`?

(Also, for the image in 2, that is what I meant for the top labels facing right and the bottom labels facing left in one of the other comments)